app/components/forms/events/view/create-access-code.js

```diff
@@ -137,8 +137,8 @@ export default Component.extend(FormMixin, {
   },
   accessCode : '',
   accessUrl  : computed('data.code', function() {
-    const params = this.get('routing.router.router.state.params');
-    return location.origin + this.get('router').urlFor('public', params['events.view'].event_id, { queryParams: { access_code: this.get('data.code') } });
+    var routeHandlers = this.get('router._router.currentState.routerJs.currentHandlerInfos');
```

Comment:
You could just use https://github.com/fossasia/open-event-frontend/issues/1166#issuecomment-391858568